src/api/app/models/obs_factory/staging_project.rb

```diff
@@ -53,6 +53,16 @@ def adi_staging?
       /#{ADI_NAME_PREFIX}/.match?(name)
     end
 
+    # @return [Boolean] true if the project is tracked in the new API
+    def part_of_workflow?
```

Comment:
`has_workflow?` or `workflow_present?` are IMO better names than `part_of_workflow?` What do you think @coolo ?

Reply:
`has_` sounds wrong - but I'm fine with `workflow_present?` if you insist

Reply:
yes please. I wasn't sure about `has_` and probably rubocop will complain about it... yes `workflow_present?` is definitely better :+1:

Reply:
if that's all..